src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -120,12 +120,13 @@
  :wallet/select-send-address
  (fn [{:keys [db]} [{:keys [address recipient stack-id start-flow?]}]]
    (let [[_ to-address]   (utils/split-prefix-and-address address)
+         sender           (get-in db [:wallet :current-viewing-account-address])
          collectible-tx?  (send-utils/tx-type-collectible?
                            (-> db :wallet :ui :send :tx-type))
          collectible      (when collectible-tx?
                             (-> db :wallet :ui :send :collectible))
          one-collectible? (when collectible-tx?
-                            (= (collectible.utils/collectible-balance collectible) 1))]
+                            (= (collectible.utils/collectible-balance collectible sender) 1))]
```

Comment:
Picking the right balance of the collectible to send.

Previously, we took the first balance in the list, which may not be the right balance of the selected account.